recipe/meta.yaml

```diff
@@ -16,6 +16,9 @@ build:
     - python -m pip install --no-deps --ignore-installed --use-feature=in-tree-build .
```

Comment:
The build is currently failing because the `--use-feature=in-tree-build` flag has been removed from `pip`. See https://github.com/conda-forge/tabula-py-feedstock/pull/24 for a pull request removing that flag. After merging that, this build should succeed again.